tests/h/services/organization_test.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+import pytest
+
+from h.models import Organization
+from h.services.organization import OrganizationService
+from h.services.organization import organization_factory
+
+
+class TestOrganizationService(object):
+    def test_create_organization_returns_organization(self, service):
+        organization = service.create_organization(
+            name='Organization',
+            authority='publisher.com',
+            logo='<svg>H</svg>')
+
+        assert isinstance(organization, Organization)
+
+    def test_create_organization_default_logo_returns_organization(self, service):
```

Comment:
I'm not certain what this is testing as the `assert` is identical to the preceding test. Is it testing that the `create` method can handle a missing `logo` param? Relevant to perhaps assert the value of the `logo` attribute?